src/PanoImageRenderer/PanoImageRenderer.js

```diff
@@ -465,9 +465,9 @@ export default class PanoImageRenderer extends Component {
 			this.context.uniform1f(this.context.getUniformLocation(this.shaderProgram, "uIsEAC"), isEAC);
 		} else if (this._imageType === ImageType.PANORAMA) {
 			const {width, height} = this._renderer.getDimension(this._image);
-			const aspectRatio = width && height && width / height;
+			const imageAspectRatio = width && height && width / height;
 
-			this._renderer.updateShaderData({aspectRatio});
+			this._renderer.updateShaderData({imageAspectRatio});
```

Comment:
if width or height is zero, imageAspectRatio is MIN_ASPECT_RATIO_FOR_FULL_PANORAMA?

The default value is set if imageAspectRatio is ```undefined```.

Reply:
@younkue There's no case height or width is zero.